src/QueryBuilder.php

```diff
@@ -728,7 +728,7 @@ protected function getDuplicatesQuery()
             ->newNestedSetQuery($firstAlias)
             ->toBase()
             ->from($this->query->raw("{$table} as {$waFirst}, {$table} {$waSecond}"))
-            ->whereRaw("{$waFirst}.{$keyName} < {$waSecond}.{$keyName}")
+            ->whereRaw("{$waFirst}.id < {$waSecond}.id")
```

Comment:
Now it replaces `{$keyName}` with `id` :confused:

Reply:
Ahhh, I'm sorry. I fixed it.